src/controller/ExampleOperationalCredentialsIssuer.h

```diff
@@ -83,6 +84,12 @@ class DLL_EXPORT ExampleOperationalCredentialsIssuer : public OperationalCredent
      */
     static CHIP_ERROR GetRandomOperationalNodeId(NodeId * aNodeId);
 
+    /**
+     * This is a utility method that generates a operational certificate chain for the given public key.
+     */
+    CHIP_ERROR GenerateNOCChainAfterValidation(NodeId nodeId, FabricId fabricId, const Crypto::P256PublicKey & ephemeralKey,
```

Comment:
What does the AfterValidation bit mean?  What is being validated?  Is this validation that this method is meant to perform or validation that this method assumes the caller has performed?  Please document.

Reply:
AfterValidation is to convey that all the checks have been made for the credentials to be generated. It’s more meaningful when this function is being used for device’s NOC Chain generation. For commissioner's NOC Chain generation, we don't need to perform device attestation/CSR etc. So those checks are skipped.

Reply:
Maybe document that this function assumes that it's OK to generate a NOC for the given node (in that any device attestation and whatever that needs to happen has happened)?